helm/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Is it possible to use the existing YAML defined without creating a new one?

Reply:
Rahul, Right now the deployment strategy we implemented is Deploying services in ArgoCD **directly from Helm artifactory**, So helm chart is mandatory for CD.